docs/contributing/setup.md

```diff
@@ -4,19 +4,30 @@
 
 You will need to install these tools on your machine:
 
- - [Nodejs](https://nodejs.org) - v7 is preferred as it's the version embedded into Electron
-    - **Windows developers**: *Make sure you allow the Node.js installer to add
-      node to the PATH, it makes life much easier.*
- - Python 2.7 - [Windows](https://www.python.org/downloads/windows/), [macOS](https://www.python.org/downloads/mac-osx/)
-    - **Windows developers**: *Let Python install into the default suggested path
-      (`c:\Python27`), otherwise you'll have to configure node-gyp manually with
-      the path which is annoying.*
- - **macOS:** Xcode and Xcode Command Line Tools (Xcode -> Preferences -> Downloads)
- - **Windows:** Visual Studio 2015 or [Visual C++ Build Tools](http://go.microsoft.com/fwlink/?LinkId=691126)
-    - Run `npm config set msvs_version 2015` after installing the build tools.
-    - *Visual Studio 2017 support has not been tested yet - waiting on
-      `node-gyp` support to be generally available.*
-
+### macOS
+
+ - [Nodejs v7](https://nodejs.org/en/download/current) - this is the version embedded into Electron
+ - [Python 2.7](https://www.python.org/downloads/mac-osx/)
+ - Xcode and Xcode Command Line Tools (Xcode -> Preferences -> Downloads)
+
+### Windows
+
+ - [Nodejs v7](https://nodejs.org/en/download/current) - this is the version embedded into Electron
+    - *Make sure you allow the Node.js installer to add node to the PATH.*
+ - [Python 2.7](https://www.python.org/downloads/windows/)
+    - *Let Python install into the default suggested path (`c:\Python27`), otherwise you'll have
+      to configure node-gyp manually with the path which is annoying.*
+    - Ensure the **Add python.exe to Path** option is selected.*
+ - Visual Studio 2015 or [Visual C++ Build Tools](http://go.microsoft.com/fwlink/?LinkId=691126)
+    - *If Visual Studio is already installed you will need to make sure you have Visual C++
+      installed as part of it. The **Common Tools for Visual C++ 2015** feature is all that's
+      required.*
+    - *Visual Studio 2017 support has not been tested yet - see [#1766](https://github.com/desktop/desktop/issues/1766) for details*
+ - From the [Windows 8.1 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-8-1-sdk) you will need the **Windows Software Development Kit** feature.
```

Comment:
I just did a run though with VS2015 Update 3 and this optional dependency:

<img width="484" alt="screen shot 2017-05-29 at 8 31 50 am" src="https://cloud.githubusercontent.com/assets/359239/26548969/ffc5a39a-444c-11e7-9dcd-b4313fd07ed4.png">

Which then resulted in this installation on top of the core:

<img width="479" alt="screen shot 2017-05-29 at 8 32 10 am" src="https://cloud.githubusercontent.com/assets/359239/26548979/0c97e060-444d-11e7-9fdd-78a3ed910930.png">

And I could `npm install` fine. Maybe this requirement relates to the VS2015 requirement for **Common Tools for Visual C++ 2015**, but I'll confirm this isn't necessary after doing a setup without VS2015 to confirm.